website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Notifications
+---
+
+In the previous sections, we learned how to develop forms. In this section we look at how to communicate feedback to users using notifications, in particular in response to user actions.
+
+To facilitate the use of notifications, Custom Applications come with a built-in notification system.
+
+# Notification system
+
+Notifications are an important way of providing feedback in response to user actions.
+For example submitting a form might succeed or fail and as a result the user should be notified about that in a way that is understandable and easy to find.
+
+Notifications are grouped by what we call "domains". A domain indicates where the notification should be displayed on the page.
+
+Notifications are also of different types: `success`, `error`, `warning`, `info`. Depending on the specific use case you would choose the appropriate type. For example, if a form has been successfully submitted, you would show a "side" notification of type `success`.
+
+Triggering notifications is done by dispatching global actions. Please refer to the package [`@commercetools-frontend/actions-global`](/api-reference/commercetools-frontend-actions-global) for implementation details.
+
+<Info>
+
+For defining the domain and the type of a notification we recommend using the values exposed from `@commercetools-frontend/constants`.
+
+For instance:
+
+```js
+import {
+  useShowNotification
+} from '@commercetools-frontend/actions-global';
+import { 
+  DOMAINS, 
+  NOTIFICATION_KINDS_SIDE,
+  NOTIFICATION_KINDS_PAGE
```

Comment:
I didn't understand why do we have different kinds per domain and have to look into the source code of `constants` packages.
It seems we are imposing some constraints on what kinds can be used in each domain.
Perhaps is this something we also want to explain?

Reply:
Yes, that's right, rather than "constraints" we call them "best practices" or "recommendations". They are already mentioned in `## Page notifications` and `## Side notifications` within `<Info>` section. Do you think that requires updating?

Reply:
In those section we recommend which kind of notification a user should use in each domain, but, as a user, I was expecting to only have one `NOTIFICATION_KINDS` constant and didn't understand why there are different ones until I saw the code and realized we have slightly different kinds per domain.
Perhaps we don't want to go into that detail, but I was confused because of that.

Maybe we can wait to have other opinions (perhaps I'm the only one confused).

Reply:
You mean these variations?

https://github.com/commercetools/merchant-center-application-kit/blob/d3166e9315b2cfcd2c00dd7f7dcd01357a9f0f2d/packages/constants/src/constants.ts#L19-L39

Yes good point, we should probably list them in each section (page and side).